SU-SWS/linky_clicky
# SITES-528: Behat test changes for JSV on socorro build

# ALMOST - READY FOR REVIEW

# Summary
- Green light the JSV build of Socorro

# Needed By (Date)
- End of June

# Urgency
- Medium

# Steps to Test

1. Check out the latest of the socorro branch
2. Run composer install
3. Start lando `lando start`
4. Install a fresh build of JSV `lando drush si stanford_sites_jumpstart install_configure_form.itasks_extra_tasks=acsf -y` 
5. Check out this branch of linky_clicky in `/vendor/su-sws/linky_clicky`
6. Run JSV behat tests `lando behat --config=/app/tests/behat/jsv/behat.yml -s all`
7. Wait about 60 minutes for the full suite to run
8. View results in terminal or at `tests.cardinald7.sws`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
